### PR TITLE
Write tests

### LINKED ISSUE #33: Add some tests
Let's add some basic tests to this. 

We can use mapbox-gl-draw as a template: https://github.com/mapbox/mapbox-gl-draw/tree/master/test.

### PR DESCRIPTION
**Pull request by tristen**
This PR:

- Adds tests
- Makes examples a little more accessible for viewing (follows a similar [_debug_ folder format in mapbox-gl-js](https://github.com/mapbox/mapbox-gl-js/tree/main/debug))
- Adds Travis for continuous integration.

I'm using [smokestack](https://www.npmjs.com/package/smokestack) for tests as a lightweight alternative to browser automation.

- Closes #33

---

cc @allthesignals @andrewharvey

### CODE REVIEW

File: test/test.js [hunk 1]
```diff
@@ -0,0 +1,105 @@
...
+
+    map.addControl(new MapboxAccessibility({
+      accessibleLabelProperty: 'name',
+      layers: [
+        'poi-label',
+        'transit-label'
+      ]
+    }));
+
+    window.setTimeout(() => {
```
Comment (tristen) [inline test/test.js:43]: This isn't ideal, but it looks like we need to wait for a beat until the accessible elements are on the map.

**Comment by tristen** [comment]
Hmm this is a little weird. Sometimes `Travis CI - Pull Request` passes/fails, sometimes `Travis CI - Branch` passes/fails. The order of operations must be off 🤔

**Comment by tristen** [comment]
I'd like to get these changes in so I am going to punt on browser automation in Travis and make it a checklist requirement in the PR template to do.

**Comment by allthesignals** [comment]
Thank you so much @tristen! I would love to revisit this and work on it some more.

**Pull request merged**